SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -152,13 +152,23 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		Connection->LegacyLocatorConfig.DeploymentName = LoadedWorld->URL.GetOption(TEXT("deployment="), TEXT(""));
 		Connection->LegacyLocatorConfig.LoginToken = LoadedWorld->URL.GetOption(TEXT("token="), TEXT(""));
 		Connection->LegacyLocatorConfig.UseExternalIp = true;
+
+		if (!GameInstance->GetSpatialWorkerType().IsEmpty())
```

Comment:
When is it valid for this to be empty?

Reply:
Hmm, there shouldn't be a case where this is valid now that we read the commandline in the game instance constructor. I wonder if we want to change the code in SpatialWorkerConnection that checks for empty strings too

Reply:
I think the SpatialWorkerConnection empty checks are fine for default setting worker type if the user hasn't passed anything into command line.

Reply:
Discussed, added a warning, let me know your thoughts